modules/ROOT/pages/tools/io-tool.adoc

```diff
@@ -0,0 +1,110 @@
+= IO-Tool
+:imagesdir: 
+
+== What is the IO-tool?
+
+https://io.my-agrirouter.com/[The agrirouter InOut-Tool] is a tool to simulate a remote endpoint and to learn to communicate via agrirouter. 
+
+image::io-tool/settings.png[]
+
+
+Using the IO-Tool you can 
+
+* exchange data
+* send and receive telemetry data
+* set preferences for the data exchange
+* check the logs and debug
+* simulate end points and set routings
+
+
+== Getting Started
+
+First you need to log in to your account, which can be requested from Agrirouter support (support@my-agrirouter.com).
+
+Here is an email template for you
+
+_Dear Sir/Madam,_
+
+_My name is (first name, last name) from (company) and I would like to request an IO-Tool account for the email address (your email). In case of any questions please reach out to (contact person)._
+
+_Thank you in advance._
+
+
+image::io-tool/login.png[]
+
+The main page of the OI-Tool is then displayed with the three menu sections 
```

Comment:
IO-Tool ;)

Reply:
We should have a sub-title here like "Initial screen and its components" or such (not perfect wording, you'll find a better one)